agent/src/main/java/com/teamscale/jacoco/agent/options/AgentOptionsParser.java

```diff
@@ -107,8 +109,9 @@ private void handleOption(AgentOptions options, String optionPart) throws AgentO
 		} else if (key.startsWith("artifactory-") && ArtifactoryConfig
 				.handleArtifactoryOptions(options.artifactoryConfig, filePatternResolver, key, value)) {
 			return;
-		}
-		if (handleAzureFileStorageOptions(options, key, value)) {
+		} else if (key.startsWith("azure-") && AzureFileStorageConfig
```

Comment:
Why does the Azure stuff show up here as changed? Seems wrong to me...

Reply:
I moved the `handleAzureFileStorageOptions` to AzureFileStorageConfig similar to how it is done for artifactory to resolve the TS file length finding 😉